frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -364,7 +364,7 @@ object Project {
       val ysemanticdb = if (!options.contains("-Ysemanticdb")) List("-Ysemanticdb") else Nil
       val sourceRoot =
         if (!options.contains("-sourceroot")) List("-sourceroot", workspaceDir.toString()) else Nil
-      ysemanticdb ++ sourceRoot
+      options ++ ysemanticdb ++ sourceRoot
```

Comment:
This is the main reason we've been having issue and a 100% my fault :sweat_smile: